src/sdk/strategy/cbd-strategy.ts

```diff
@@ -0,0 +1,126 @@
+import { ethers } from 'ethers';
+
+import { CbdTDecDecrypter } from '../../characters/cbd-universal-bob';
```

Comment:
Reconsider the name `cbd-universal-bob.ts`. At the very least, we should drop the `universal` (which made sense in the context of fake, PRE-based TDEC), although I'm much in favour in getting rid of character names, so perhaps something like `cbd-consumer` or `cbd-recipient`.